src/core.js

```diff
@@ -458,7 +458,7 @@ function( i, name ) {
 
 function isArrayLike( obj ) {
 
-	// Support: iOS 8.2 (not reproducible in simulator)
+	// Support: iOS 8.2 only (not reproducible in simulator)
```

Comment:
I think this would be better as `Support: real iOS 8.2 only` or `Support: non-simulated iOS 8.2 only`.